### PR TITLE
feat(pci-kubernetes): add TAPC changes to reset kube modal

### PR DESCRIPTION
**Pull request by seven-amid**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/pci-kubernetes
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-*
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-kubernetes/src/components/network/SubnetSelector.component.tsx [hunk 1]
```diff
@@ -72,52 +73,58 @@ export const SubnetSelector = ({
...
+          {allowsEmpty && (
+            <OsdsSelectOption key="none" value="none">
+              {tCommon('common_none')}
+            </OsdsSelectOption>
+          )}
+          {availableSubnets?.map((sub) => (
+            <OsdsSelectOption key={sub.id} value={sub.id}>
+              {sub.id}
+              {' - '}
+              {sub.cidr}
```
Comment (anooparveti) [inline packages/manager/apps/pci-kubernetes/src/components/network/SubnetSelector.component.tsx:121]:
```suggestion
              {`${sub.id} - ${sub.cidr}`}
```

File: packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx [hunk 2]
```diff
@@ -69,17 +78,31 @@ export default function ResetClusterPage() {
     isPending: isPendingPrivateNetworks,
   } = useAvailablePrivateNetworks(projectId, kubernetesCluster?.region);
 
+  console.log({ privateNetworks });
```
Comment (anooparveti) [inline packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx:81]: **nitpick:** Remove console statement.
```suggestion
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx [hunk 3]
```diff
@@ -69,17 +78,31 @@ export default function ResetClusterPage() {
...
+
+  const defaultNetwork = {
+    id: 'none',
+    name: t('kubernetes_network_form_none'),
+  } as TNetwork;
+
+  const getPrivateNetworkById = (id: string): TNetwork =>
+    privateNetworks?.find((net) => net.clusterRegion.openstackId === id);
```
Comment (anooparveti) [inline packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx:89]:
```suggestion
    privateNetworks?.find((privateNetwork) => privateNetwork.clusterRegion.openstackId === id);
```

File: packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx [hunk 4]
```diff
@@ -69,17 +78,31 @@ export default function ResetClusterPage() {
...
+
   const [formState, setFormState] = useState({
     workerNodesPolicy: WORKER_NODE_POLICIES.DELETE,
     selectedVersion: getFormatedKubeVersion(kubernetesCluster?.version || ''),
     privateNetworkId: kubernetesCluster?.privateNetworkId,
+    subnet: undefined,
+    loadBalancersSubnet: undefined,
```
Comment (anooparveti) [inline packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx:96]: Use `null` instead.
```suggestion
    subnet: null,
    loadBalancersSubnet: null,
```

**Review by sidlynx — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Pull request merged**